doc/040_backup.rst

```diff
@@ -586,6 +586,9 @@ environment variables. The following lists these environment variables:
 
     TMPDIR                              Location for temporary files
 
+    REST_USERNAME                       REST Server username
+    REST_PASSWORD                       REST Server password
```

Comment:
I looked at existing variables, and it seems that most of them start with the backend name (eg. `AWS_ACCESS_KEY_ID`, `B2_ACCOUNT_ID`, `AZURE_ACCOUNT_NAME`, etc), so I'd say not adding `RESTIC_` would be more consistent, but I can change it if it's likely to collide with existing variables

Reply:
If I am not wrong we are talking about 'Restic REST' server right? 

Keeping that in mind `RESTIC_REST_` seems like a better option to me, since we are following the concept of prefixing the backend type. 

I think `REST` is a very generic prefix.

Not important just my opinion.

Reply:
I agree that `REST` is extremely generic. The other environment variables match the canonical names used by the corresponding cloud providers. However, rest-server only makes sense in the context of restic, such that a `RESTIC_REST_` is preferable. Please update the environment variable name.

Btw. please move the variables to the end of the list. Even though the current ordering is a complete chaos, I'd prefer to not further deviate from an alphabetical order. (Feel free to reorder the variable names such that their prefixes are sorted alphabetically. But please do not change the order within e.g. the AWS block).

Reply:
I ended up leaving the non-backend-specific variables at the top, but the rest should now be sorted alphabetically. I've also added the `RESTIC_` prefix.